app/views/projects/_edit_form.html.erb

```diff
@@ -25,7 +33,15 @@
       </div>
       <div class="col-md-9">
          <% if !@project.persisted? || @project.metadata_model.data_sponsor == current_user.uid %>
-         <input type="text" list="manager-users" id="data_manager" aria-label="data manager" name="data_manager" required value="<%= @project.metadata_model.data_manager %>" /></input>
+         <input type="text" list="manager-users" id="data_manager" aria-label="data manager" name="data_manager" required oninvalid="this.setCustomValidity('')"
+               oninput="this.setCustomValidity('')" value="<%= @project.metadata_model.data_manager %>" /></input>
+         <div class="custom_error" id="manager_error" aria-live="polite">
+            <svg xmlns="http://www.w3.org/2000/svg" width="25" height="24" viewBox="0 0 25 24" fill="none">
```

Comment:
I wonder if there is a way to extract this svg so it does not need to be in each location.  Assuming it is the same thing.   That said, this should not block the PR.